watertap/unit_models/translator_adm1_asm1.py

```diff
@@ -0,0 +1,436 @@
+**False** - outlet property package should enforce sumation and equilibrium
+constraints.}""",
+        ),
+    )
+    CONFIG.declare(
+        "inlet_property_package",
+        ConfigValue(
+            default=None,
+            domain=is_physical_parameter_block,
+            description="Property package to use for incoming stream",
+            doc="""Property parameter object used to define property
+calculations for the incoming stream,
+**default** - None.
+**Valid values:** {
+**PhysicalParameterObject** - a PhysicalParameterBlock object.}""",
+        ),
+    )
+    CONFIG.declare(
+        "inlet_property_package_args",
+        ConfigBlock(
+            implicit=True,
+            description="Arguments to use for constructing property package "
+            "of the incoming stream",
+            doc="""A ConfigBlock with arguments to be passed to the property
+block associated with the incoming stream,
+**default** - None.
+**Valid values:** {
+see property package for documentation.}""",
+        ),
+    )
+    CONFIG.declare(
+        "outlet_property_package",
+        ConfigValue(
+            default=None,
+            domain=is_physical_parameter_block,
+            description="Property package to use for outgoing stream",
+            doc="""Property parameter object used to define property
+calculations for the outgoing stream,
+**default** - None.
+**Valid values:** {
+**PhysicalParameterObject** - a PhysicalParameterBlock object.}""",
+        ),
+    )
+    CONFIG.declare(
+        "outlet_property_package_args",
+        ConfigBlock(
+            implicit=True,
+            description="Arguments to use for constructing property package "
+            "of the outgoing stream",
+            doc="""A ConfigBlock with arguments to be passed to the property
+block associated with the outgoing stream,
+**default** - None.
+**Valid values:** {
+see property package for documentation.}""",
+        ),
+    )
+
+    def build(self):
+        """
+        Begin building model.
+        Args:
+            None
+        Returns:
+            None
+        """
+        # Call UnitModel.build to setup dynamics
+        super(TranslatorData, self).build()
+
+        self.N_I = Param(
+            initialize=0.06
+            / 14,  # change from 0.002 to 0.06/14 based on Rosen & Jeppsson, 2006
+            units=pyunits.kmol * pyunits.kg**-1,
+            mutable=True,
+            doc="Nitrogen content of inerts [kmole N/kg COD]",
+        )
+        self.N_aa = Param(
+            initialize=0.007,
+            units=pyunits.kmol * pyunits.kg**-1,
+            mutable=True,
+            doc="Nitrogen in amino acids and proteins [kmole N/kg COD]",
+        )
+        self.N_bac = Param(
+            initialize=0.08 / 14,
+            units=pyunits.kmol * pyunits.kg**-1,
+            mutable=True,
+            doc="Nitrogen content in bacteria [kmole N/kg COD]",
+        )
+
+        self.i_ec = Param(
```

Comment:
@adam-a-a this may belong in the parameter_block. Any thoughts?

Reply:
Where is this value from? I didn't see it in the Rosen & Jeppsson paper...unless it's just the numerator of N_I

Reply:
from Copp simulation manual pp6 called i_XP